test/lib/specHelperSpec.js

```diff
@@ -108,5 +115,19 @@ describe('Fluxxed up test helpers', function() {
       // Signal that test should be finished:
       rig.finish()
     })
+
+    it('toggles the radio button', () => {
+      var element = rig.domNode.find('.radio-input')[0]
+      rig.toggleRadioButton(element)
+
+      expect(rig.domNode.find('.radio-input')[0].checked).to.equal(true)
+    })
+
+    it('selects the right value for the select box', () => {
+      var element = rig.domNode.find('.select-input')[0]
+      rig.select(element, "1")
```

Comment:
@andela-tkomolafe And this can become `rig.setValue('.select-input', '1')`. (You could also set `select` as an alias of `setValue` if you want that `type`-specific method to still exist)